<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWidget</class>
 <widget class="QWidget" name="UpdateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="1">
    <widget class="QPushButton" name="pushButtonImport">
     <property name="text">
      <string>导入设备列表</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonSelAll">
     <property name="text">
      <string>全选</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QTableWidget" name="tableWidgetDevList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>升级状态</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButtonStart">
     <property name="text">
      <string>开始升级</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBoxUpdateFile">
     <property name="title">
      <string>升级文件</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>包大小</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxPacket">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>64</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>128</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>248</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>升级类型</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="comboBoxUpdateType">
        <item>
         <property name="text">
          <string>适配层</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>网络层</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>应用层</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pushButtonGetFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEditPath"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="5" rowspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>文件信息</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTextBrowser" name="textBrowserFileInfo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonDeSel">
     <property name="text">
      <string>反选</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
